includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -11,6 +11,7 @@
 
 use WCPay\Exceptions\{ Add_Payment_Method_Exception, Process_Payment_Exception, Intent_Authentication_Exception, API_Exception };
 use WCPay\Logger;
+use WCPay\MultiCurrency\MultiCurrency;
```

Comment:
We finally don't need to use `MultiCurrency` here, right?

Reply:
Good catch! This was leftover from a previous attempt. Fixed in 3795b84d02b8a7b3758e28e063426feecba97a30!